cluster-autoscaler/cloudprovider/azure/azure_cloud_provider.go

```diff
@@ -75,7 +75,7 @@ func (azure *AzureCloudProvider) Name() string {
 
 // GPULabel returns the label added to nodes with GPU resource.
 func (azure *AzureCloudProvider) GPULabel() string {
-	return legacyGPULabel // Use legacy to avoid breaking, for now
```

Comment:
`legacyGPULabel` is still currently used in azure_template.go to scaffold new Nodes. I elected to keep that since it doesn't seem like it would hurt anything going forward and would be one less blocker in case users try to downgrade cluster-autoscaler from 1.32 to an older version.